2020-02-Spring/1_strings/README.md

```diff
@@ -1,7 +1,7 @@
 <!-- Don't remove -->
 <a name="top"/>
 
-# Topic Name TODO :bug:
+# Strings
```

Comment:
don't need to do anything at this point but I changed the title name in the first PR. I can resolve any conflicts later.